### PR TITLE
chore: remove dnf cache and unused files from built image

### PR DESCRIPTION
**Pull request by p5**
We should remove dnf cached files from the image before publishing it.
Also we don't need to include files like the devcontainer config or action.yml in the image, so only including the necessary files.

### CODE REVIEW

File: Containerfile [hunk 1]
```diff
@@ -14,10 +14,14 @@ ENV VARIANT="Kinoite"
 ENV VERSION="${VERSION}"
 ENV WEB_UI="false"
 
-COPY / /isogenerator
+COPY ./lorax_templates /isogenerator/lorax_templates
+COPY ./xorriso /isogenerator/xorriso
+COPY ./Makefile /isogenerator/
```
Comment (noelmiller) [inline Containerfile]: You will need to add ./scripts now that it is a dependency for secure boot.

**Review by noelmiller — CHANGES_REQUESTED** [review]

File: Containerfile (hunk 1 above)
Comment (p5) [reply Containerfile]: [302a52c](https://github.com/ublue-os/isogenerator/pull/26/commits/302a52c100ebfd848129311da3d6ef58aa48dcf9)

**Review by JasonN3 — DISMISSED** [review]

**Comment by noelmiller** [comment]
`entrypoint.sh` also needs to be copied over. I'm starting to think saving space on the `action.yml`, `LICENSE file`, `README.md`, and `Containerfile` isn't worth the hassle of continuing to add files to the list of things copied into the container.

@bsherman thoughts?

**Comment by p5** [comment]
> `entrypoint.sh` also needs to be copied over. I'm starting to think saving space on the `action.yml`, `LICENSE file`, `README.md`, and `Containerfile` isn't worth the hassle of continuing to add files to the list of things copied into the container.

Have switched to a .dockerignore instead.  That way, we are including files by default, instead of the previous way of excluding them by default.

**Review by noelmiller — APPROVED** [review]

**Comment by p5** [comment]
:facepalm: I cancelled the builds in the wrong tab.  Will rerun them.  
Sorry

**Review by bsherman — APPROVED** [review]

**Pull request merged**